### PR TITLE
Add skipped event listener for allure

### PR DESCRIPTION
**Pull request by koushikmohan1996**
## Motivation/Description of the PR

Applicable plugins:

- [X] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [X] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/plugin/allure.js [hunk 1]
```diff
@@ -234,6 +234,18 @@ module.exports = (config) => {
     reporter.endCase('passed');
   });
 
+  event.dispatcher.on(event.test.skipped, (test) => {
+    let loaded = true;
+    if (reporter.getCurrentSuite() !== test.parent.fullTitle()) {
```
Comment (DavertMik) [inline lib/plugin/allure.js:239]: please update to use `if (test.isFastSkipped)` or something like this

Comment (koushikmohan1996) [reply lib/plugin/allure.js]: Added to skip info `test.opts.skipInfo.isFastSkipped`

**Comment by Georgegriff** [comment]
Im seeing flakey unit tests due to this change in allure_test.js

It appears that two test reports are now generated on failure:
image
Depending on which hook gets called first, this new one, or existing one in allure.js then the tests fail

@DavertMik @koushikmohan1996

**Comment by Georgegriff** [comment]
Update: Looks like the two changes work nicely together, i'll see about fixing the flakey tests 
image

**Pull request merged**